src/sentry/api/urls.py

```diff
@@ -1210,6 +1214,21 @@
                     OrganizationStatsEndpoint.as_view(),
                     name="sentry-api-0-organization-stats",
                 ),
+                url(
```

Comment:
need to decide on how we do versioning with this API. we intend to keep the old one around in case some customers are using it to generate reports. for now, `_v2` is added there.

Reply:
Hi! Long time listener, first time caller. Can we check the logs to confirm anyone outside Sentry is directly calling the original API?

Reply:
looks like 11 orgs have called this API in the last 90 days in sentry SaaS using the best proxy i could for non-sentry-webpage calls.